src/builder/QuarkBuilderHelper.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.23;
+
+import {BridgeRoutes} from "./BridgeRoutes.sol";
+
+/**
+ * @title Quark Builder Helper
+ * @notice External-facing functions that might be helpful for those using the QuarkBuilder
+ */
+contract QuarkBuilderHelper {
+    /* ===== Constants ===== */
+
+    uint256 constant PAYMENT_COST_BUFFER = 1.2e18;
+    uint256 constant BUFFER_SCALE = 1e18;
+
+    function canBridge(uint256 srcChainId, uint256 dstChainId, string memory assetSymbol)
+        external
+        pure
+        returns (bool)
+    {
+        return BridgeRoutes.canBridge(srcChainId, dstChainId, assetSymbol);
+    }
+
+    function addBufferToPaymentCost(uint256 maxPaymentCost) external pure returns (uint256) {
+        return maxPaymentCost * PAYMENT_COST_BUFFER / BUFFER_SCALE;
+    }
```

Comment:
Looks like it's based on constant, and always just add 20% on top of the maxParymentCost? 🤔

Reply:
Correct, that's the heuristic we use for now.